src/libcore/marker.rs

```diff
@@ -166,6 +166,11 @@ pub trait Unsize<T: ?Sized> {
 /// are allowed to access `x` after the assignment. Under the hood, both a copy and a move
 /// can result in bits being copied in memory, although this is sometimes optimized away.
 ///
+/// ## Closures
+///
+/// Closure types automatically implement `Clone` if they capture no value from the environment
```

Comment:
> `Clone`

is that a typo for `Copy`?

Reply:
Oops, hasty copy-paste. Good catch, thanks. I just squashed a fix.